Bottom: e16b6dec36e43ba2217ba4904527865e38e71926
Top:    fcb939492941fdc54d1ffa413b4a39b976d4aebb
Author: Andrew Cooper <andrew.cooper@novatechweb.com>
Date:   2018-08-15 09:47:05 -0500

Refresh of worker-configuration

---
diff --git a/docker/buildbot-master/buildbot/openembedded.py b/docker/buildbot-master/buildbot/openembedded.py
index 4a67093b..c946d579 100755
--- a/docker/buildbot-master/buildbot/openembedded.py
+++ b/docker/buildbot-master/buildbot/openembedded.py
@@ -38,14 +38,6 @@ c['workers'] = [
 
 DEFAULT_REPO = 'git@git.novatech-llc.com:ntel/setup-scripts.git'
 
-c = WorkerConfig = {}
-
-
-# BUILDERS
-# The 'builders' list defines the Builders, which tell Buildbot how to perform
-# a build: what steps, and which workers can execute them.  Note that any
-# particular build will only take place on one worker.
-c['builders'] = []
 
 # CHANGESOURCES
 c['change_source'] = [
